src/main/scala/dotty/dokka/tasty/SyntheticSupport.scala

```diff
@@ -33,4 +33,10 @@ trait SyntheticsSupport:
     given dotc.core.Contexts.Context = r.rootContext.asInstanceOf
     val sym = rsym.asInstanceOf[dotc.core.Symbols.Symbol]
     sym.is(dotc.core.Flags.Extension)
-  }
\ No newline at end of file
+  }
+
+  def getExtendedSymbol(d: Symbol): Option[ValDef] = 
+    Option.when(hackIsExtension(self.reflect)(d))(
+      if(d.name.endsWith(":")) d.tree.asInstanceOf[DefDef].paramss(1)(0)
```

Comment:
there should be a `.isLeftAssoc` method exposed in Tasty-Reflect but we forgot to do it, can we change this line to something line `hackIsLeftAssoc(d)` and define the method appropriately? The idea is that whenever we upgrade to a new version of Tasty, we inspect all the `hack*` methods and see if we can replace them with Tasty-Reflect calls.

Reply:
Done.